core/src/builtins/mod.rs

```diff
@@ -83,3 +88,7 @@ fn queue_microtask(
     let func = v8::Local::<v8::Function>::try_from(obj).unwrap();
     scope.enqueue_microtask(func);
 }
+
+fn url(scope: &mut v8::HandleScope, args: v8::FunctionCallbackArguments, _rv: v8::ReturnValue) {
+    
```

Comment:
I sorta understand things up to this point. Would love some input here as I try to hack something together.